generators/client/templates/_package.json

```diff
@@ -11,6 +11,7 @@
     "counterpart": "^0.17.2",
     "deep-equal": "^1.0.1",
     "flexboxgrid": "^6.3.1",
+    "fixed-data-table-2": "^0.7.12",
```

Comment:
While the library is good, I dont think we need most of the features for our generated code. We use server side pagination and sorting. I was thinking of using http://www.material-ui.com/#/components/table